components/layout/default.tsx

```diff
@@ -572,8 +572,9 @@ const DefaultLayout: React.FC<{ children: React.ReactNode }> = ({
             field: 'featureFlags',
             value: {
               // Ignore feature flag on staging and local env
-              'enable-chat-with-doc': isFeatureEnabled('enable-chat-with-doc') || process.env.NEXT_PUBLIC_ENV !== 'production',
-              'enable-conversation-mode': isFeatureEnabled('enable-conversation-mode') || process.env.NEXT_PUBLIC_ENV !== 'production',
+              'enable-conversation-mode':
+                isFeatureEnabled('enable-conversation-mode') ||
```

Comment:
@1orZero I assume this `isFeatureEnabled` also needs to be removed?

Reply:
Should I  remove all the feature flags so that the `enable-conversation-mode` feature is open to everyone?

Reply:
Yepp that would be ideal. Open to everyone who has Ultra plan